enterprise/frontend/src/metabase-enterprise/whitelabel/components/LandingPageWidget/LandingPageWidget.tsx

```diff
@@ -33,7 +30,7 @@ export function LandingPageWidget({ onChangeSetting, settingValues }: Props) {
     } else {
       setError(null);
       try {
-        await onChangeSetting("landing-page", relativeUrl);
+        await onChange(relativeUrl);
```

Comment:
I guess `onChangeSetting` wasn't meant to be used by the main setting as it never calls the callback that renders the setting saved notification toast.

![image](https://github.com/metabase/metabase/assets/1937582/2b3ea3b7-2f5a-4492-aa01-c438365d6c0d)